OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -92,6 +92,12 @@ internal static void TryUpdateHSLColor(ref string value)
 			catch { }
 		}
 
+		internal static string MultiplyByFactor(int oldValue, int factor)
```

Comment:
Deciding to split only this part into a function seems like a really weird choice to me.

Please either inline this as one line (`= (oldValue * 10).ToString()`) or move the `var oldValue` and the mod id checks inside here to properly remove the duplication.